metrics/src/metrics.rs

```diff
@@ -0,0 +1,212 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+
+use ddcommon::tag::{parse_tags, Tag};
+use serde::Serialize;
+
+#[derive(PartialEq, Debug, Serialize, Clone)]
+pub struct Points {
+    pub timestamp: u64,
+    pub values: Vec<f64>,
+}
+
+impl Points {
+    pub fn new(values: Vec<f64>, timestamp: u64) -> Self {
+        Self { timestamp, values }
+    }
+}
+
+#[derive(PartialEq, Debug, Serialize, Clone)]
```

Comment:
There's a lot of different metrics types, but we only care about distribution metrics for now.